DRAFT.md

```diff
@@ -715,6 +715,25 @@ constexpr indirect<T, Alloc>& operator->() noexcept;
 
 Otherwise, the interface of the specialization is as defined in [optional].
 
+#### X.Y.13 Formatter support [indirect.fmt]
+
+```c++
+// [indirect.fmt]
+template <class T, class Alloc, class charT>
+struct std::formatter<indirect<T, Alloc>, charT> : std::formatter<T> {
+  constexpr typename FormatContext::iterator parse(format_parse_context& ctx);
```

Comment:
I only have limited experience with `std::format`, but I think there are two issues in this line.
1. `FormatContext` gets used but I don't see where it is coming from
2. You support any `class charT`, but `parse` only takes a `format_parse_context` (which is an alias to `basic_format_parse_context<char>`.
Maybe the correct syntax would be: 
```c++
constexpr typename basic_format_context<charT>::iterator parse(basic_format_parse_context<charT>& ctx);
```
or if you take the `ParseContext` as template argument (as e.g. the [range_formatter](http://eel.is/c++draft/format.range#formatter) does):
```c++
template<class ParseContext>
constexpr typename ParseContext::iterator parse(ParseContext& ctx);
```
As your `format`-function takes the `FormatContext` as template argument, I guess it's consistent to also let the `parse`-function take its `ParseContext` as template argument, right?

Reply:
Thanks, very limited experience with `std::format` yet so appreciate the feedback!